components/report/cell-renderers.js

```diff
@@ -364,6 +364,50 @@ Url.propTypes = {
   source: PropTypes.object.isRequired,
 };
 
+/**
+ * Display a attachment-download button along with the full download path to the document or image.
+ */
+function AttachmentHref({ source }) {
+  // Wrap in a div because the cell has a flex class we don't want to inherit.
+  return (
+    source.attachment?.href && (
+      <div>
+        <div className="flex">
+          <FileDownload file={source} className="shrink" />
+        </div>
+        <div>{`${API_URL}${source["@id"]}${source.attachment.href}`}</div>
+      </div>
+    )
+  );
+}
```

Comment:
You could _almost_ justify refactoring this so that `AttachmentHref` and the `Href` component above both call the same shared component, and parameterize the href. But given how small this code is, it’s not a _clear_ benefit. Up to you.